frontend/src/metabase/public/containers/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestion.unit.spec.tsx

```diff
@@ -76,13 +88,24 @@ async function setup() {
     {
       storeInitialState: createMockState(),
       withRouter: true,
-      initialRoute: `public/question/${FAKE_UUID}`,
+      initialRoute: `public/question/${FAKE_UUID}${
+        queryString ? `?` + queryString : ""
+      }${hash ? "#" + hash : ""}`,
```

Comment:
nit: I wonder if there is a function to construct a string with URL and hash enabled, this seems to be a very common pattern

Reply:
I'm not a fan of that logic but I don't think there's anything built in node that's easier to use, I'd keep the tests code as simple as possible